src/components/Navbar.js

```diff
@@ -1,53 +1,86 @@
-import styled from "styled-components"
+import styled from "styled-components";
+import { SiRiotgames } from "react-icons/si";
 
 const Navbar = () => (
-    <NavbarWrapper>
+  <NavbarWrapper>
+    <NavLogoContainer>
+      <a href={""}>
         <Logo />
-        <NavItemsContainer>
-            <NavItem>
-                <NavLink>
-                    Home
-                </NavLink>
-            </NavItem>
-            <NavItem>
-                <NavLink>
-                    About
-                </NavLink>
-            </NavItem>
-            <NavItem>
-                <NavLink>
-                    Contact
-                </NavLink>
-            </NavItem>
-        </NavItemsContainer>
-    </NavbarWrapper>
-)
-
-const Logo = () => <div>👻</div>
+      </a>
+    </NavLogoContainer>
+    <NavItemsContainer>
+      <NavItem>
+        <NavLink href={""}>Home</NavLink>
+      </NavItem>
+      <NavItem>
+        <NavLink href={""}>About</NavLink>
+      </NavItem>
+      <NavItem>
+        <NavLink href={""}>Products</NavLink>
+      </NavItem>
+      <NavItem>
+        <NavLink href={""}>Contact</NavLink>
+      </NavItem>
+    </NavItemsContainer>
+    <NavItem>
+      <Button href={""}>Sign Up</Button>
+    </NavItem>
+  </NavbarWrapper>
+);
+
+const Button = styled.button`
```

Comment:
don't create local components with the same name as shared components, can be a mess